Lib/turtle.py

```diff
@@ -1175,7 +1175,7 @@ def _color(self, cstr):
             cl = [16*int(cstr[h], 16) for h in cstr[1:]]
         else:
             raise TurtleGraphicsError("bad colorstring: %s" % cstr)
-        return tuple(c * self._colormode/255 for c in cl)
+        return tuple([c * self._colormode/255 for c in cl])
```

Comment:
The list comprehension is not necessary and doesn't improve the code. This should stay as-is.